src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml

```diff
@@ -2,23 +2,27 @@
 
 jobs:
 - job: Source_Build_Create_Tarball
-  container: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-38
+  container: fedora38
```

Comment:
This should be reverted.

Reply:
It won't function without this change. String-based container declaration doesn't work with 1ES PT.